eth2/types/src/attestation_data_and_custody_bit.rs

```diff
@@ -30,6 +30,16 @@ impl Decodable for AttestationDataAndCustodyBit {
     }
 }
 
+impl TreeHash for AttestationDataAndCustodyBit {
+    fn hash_tree_root(&self) {
```

Comment:
This doesn't seem to return anything. I thought that mean it wouldnt compile nor pass tests? Am I missing something?

Reply:
Good pickup, I guess the compiler must be smart enough to know what the return type is without having to specifically state it due to the trait implementation.

This is meant to have the return type Vec<u8> so I'll add it in